src/app/checkin/[id]/page.tsx

```diff
@@ -14,6 +11,7 @@ import {
   AlertDescription,
   AlertTitle,
 } from "@/components/ui/alert"
+import { Badge, TextInput, Button } from "flowbite-react"
```

Comment:
Was there a reason why we switched back to flowbite? When I designed this website in mind, I wanted to go for more of a minimal look and tried keeping styles consistent as possible using the UI library that was being used (https://ui.shadcn.com/)

Reply:
Mainly for consistency with Dark Mode support, I tried to only switch it for a few things, and am good with switching back if you feel strongly. But I kept the styles very similar to the custom components you were using, and think that Flowbite actually makes it easier to have a more consistent look and feel.

Lmk what you think!